.ldrelease/config.yml

```diff
@@ -1,6 +1,10 @@
+version: 2
 
-template:
-  name: go
+jobs:
+  - docker:
+      image: golang:1.14-buster
```

Comment:
Minor: as this library is compatible back to Go 1.13, shouldn't it be released with the 1.13?